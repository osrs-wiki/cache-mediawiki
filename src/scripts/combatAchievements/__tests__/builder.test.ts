import combatAchievementPageBuilder from "../builder";

describe("combatAchievementPageBuilder", () => {
  test("it should build with all fields", () => {
    const builder = combatAchievementPageBuilder({
      id: 1,
      title: "Testing",
      description: "Testing a Description",
      monster: "Big Monster",
      type: "Kill Count",
      tier: "Elite",
    });

    expect(builder.build()).toMatchSnapshot();
  });
});
