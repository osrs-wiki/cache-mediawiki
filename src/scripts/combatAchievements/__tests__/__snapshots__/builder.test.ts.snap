// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`combatAchievementPageBuilder it should build with all fields 1`] = `
"{{Infobox Combat Achievement
|name = Testing
|image = [[File:Big Monster.png|150px]]
|members = Yes
|description = Testing a Description
|tier = Elite
|monster = Big Monster
|type = Kill Count
|id = 1
}}
'''Testing''' is an [[Combat Achievements/Elite|elite combat achievement]] which requires the player to testing a Description

{{Combat Achievements}}
"
`;
